test.js

```diff
@@ -8,10 +8,10 @@ const fixtureErr = new Error('fixture');
 test('retries', async t => {
 	let i = 0;
 
-	const ret = await m(async attempts => {
+	const ret = await m(async attemptNo => {
```

Comment:
Changing `attempts` to `attemptNo` seems rather arbitrary.

Reply:
This change was deliberate: `attempts` is actually the current attempt number, according to `retry`'s docs, and not the total number of attempts as the variable name implies. It was not obvious to me when I first start developing this feature, that is why I made the change to be more clear.